### ISSUE TITLE
Missing library file for external library

### ISSUE THREAD

**Issue by stonerzju** [nokia/RED#264]
Hi-
    I tried to include an external library SSHLibrary in RED but always have issue. I tried to look around in google and issue history but can not find a solution.

I have the following instruction:
1. On windows 7 system, I have python 3.5.4 installed at  C:\Users\sesa429027\AppData\Local\Programs\Python\Python35\python.exe
2. I install package robotframework-SSHLibrary
3. I create workspace at location C:\Users\sesa429027
4. I create project testrobot, create test suite test.robot and have following code
```*** Settings ***
Library    SSHLibrary    

*** Test Cases ***
first test case
    Log  hello```
5. right click on project testrobot->robot framework->automatically discover and add library to red.xml
6. Finally, library SSHLibrary is added with relative path AppData\Local\Programs\Python\Python35\Lib\site-packages

Issue found:
In red.xml, it report `Missing library file for external library`. 
on test case editing page, everything looks fine, the auto completion also works.

Did i do something wrong?

**Comment by michalanglart** [comment]
Hi,

> `Missing library file for external library`

Is this the exact message? Where is this message shown? Is this a marker visible in `Problems` view? Can you share some screenshot?

**Comment by michalanglart** [comment]
Ok, we were able to reproduce this issue on our side.

**Comment by stonerzju** [comment]
Thanks for responding. I have workaround which generate the libspec file and import it directly in red.xml

**Comment by michalanglart** [comment]
Fixed in 0.8.11
